openapi/components/schemas/AmlCheck.yaml

```
@@ -49,11 +54,13 @@ properties:
     readOnly: true
     enum:
       - pending-review
+      - in-review
       - no-match
       - confirmed-match
       - false-positive
     x-enumDescriptions:
       pending-review: Possible AML match detected and waiting to be manually reviewed.
+      in-review: Possible AML match manual review process started.
```

suggestion: for readability
```suggestion
      in-review: A manual AML match review is in process.
```